src/http/server/server.cr

```diff
@@ -160,19 +164,11 @@ class HTTP::Server
     end
   end
 
+  # Builds all handlers as the middleware for HTTP::Server.
   def self.build_middleware(handlers, last_handler = nil : Request -> Response)
-    if handlers.empty?
-      raise ArgumentError.new "no handlers specified"
-    end
-
-    0.upto(handlers.size - 2) do |i|
-      handlers[i].next = handlers[i + 1]
-    end
-
-    if last_handler
-      handlers.last.next = last_handler
-    end
-
+    raise ArgumentError.new "You need specify at least 1 HTTP Handler." if handlers.empty?
+    0.upto(handlers.size - 2) { |i| handlers[i].next = handlers[i + 1] }
+    handlers.last.next = last_handler if last_handler
```

Comment:
What about "You must specify at least one HTTP handler." ?

I think I prefered the indented version instead of oneliners, but it's just a matter of taste. It took me some time to notice what actualy changed.